typings.d.ts

```diff
@@ -82,6 +84,10 @@ export function emptyCell(cell: ContentNodeWithPos, schema: Schema): (tr: Transa
 
 export function addColumnAt(columnIndex: number): (tr: Transaction) => Transaction;
 
+export function moveRow(originRowIndex: number, targetRowIndex: number, options?: MovementOptions): (tr: Transaction) => Transaction;
+
+export function moveColumn(originColumnIndex: number, targetColumnIndex: number, options?: MovementOptions): (tr: Transaction) => Transaction;
+
```

Comment:
Since you're exporting `convertTableNodeToArrayOfRows` and `convertArrayOfRowsToTableNode`, it'd be nice to have typings added here. Otherwise remove the exports.

Reply:
good point, I dunno why it's not there since I added the comments on the function, I'll add it manually.